cla_public/templates/macros/form.html

```diff
@@ -13,7 +13,7 @@
     - attrs <object> (default: None)
         Custom HTML attributes for fieldset element
 #}
-{% macro fieldset(field=None, class_='govuk-fieldset ', legend=None, field_attrs={}) %}
+{% macro fieldset(field=None, class_='govuk-fieldset ', legend=None, field_attrs={}, hidden_legend_prefix=None) %}
```

Comment:
`hidden_legend_prefix=None`

Why is this necessary? Wouldn't the same thing happen if we left it out?

Reply:
No, this is needed.  It is a new argument in the macro that we need.  We feed in a value there on the property pages telling it what the prefix needs to be.  If nothing is received, it defaults to none.